config.libsonnet

```diff
@@ -29,7 +29,7 @@
     podLabel: 'pod',
     hostNetworkInterfaceSelector: 'device!~"veth.+"',
     hostMountpointSelector: 'mountpoint="/"',
-    wmiExporterSelector: 'job="wmi-exporter"',
+    windowsExporterSelector: 'job="kubernetes-windows-exporter"',
```

Comment:
Is it a standard job name or should we stick to previous job name(wmi-exporter) for compatibility?

Reply:
if I'm not mistaken, `wmi-exporter` is the old exporter which has been renamed to windows exporter. The [original repo for wmi-exporter](https://github.com/martinlindhe/wmi_exporter) redirects to [windows-exporter](https://github.com/prometheus-community/windows_exporter). 

Therefore: not sure what to do, but because windows-exporter is more updated, I'd say renaming to windows exporter is fine. The selector itself depends a lot on the scrape config of course, not sure what most people use TBH.